internal/apiserver/common.go

```diff
@@ -44,6 +44,8 @@ var (
 	// ErrDBContainerNotFound is an error that indicates that a "database" container
 	// could not be found in a specific pod
 	ErrDBContainerNotFound = errors.New("\"database\" container not found in pod")
+	// ErrLabelInvalid indicates that a label is invalid
+	ErrLabelInvalid = errors.New("invalid label")
```

Comment:
This looks like a copy of `internal/util.ErrLabelInvalid`. Is it used?

Reply:
Oh I forgot to remove that during the refactor. I can handle that.